cf-stacks/compute-stack.yml

```diff
@@ -93,6 +95,7 @@ Resources:
         - 'arn:aws:iam::aws:policy/AmazonCognitoPowerUser'
         - 'arn:aws:iam::aws:policy/AmazonEventBridgeFullAccess'
         - 'arn:aws:iam::aws:policy/AmazonSQSFullAccess'
+        - 'arn:aws:iam::aws:policy/AmazonSESFullAccess'
```

Comment:
In the future, put the changes impacting CI and CD file configuration into specific commits.